Justfile

```diff
@@ -48,22 +50,26 @@ copy-assets:
   mkdir -p ./release
   cp -a assets/. release/
 
-# Generate the vcluster images file
+# Generate the vcluster latest/minimal images file
+[private]
+generate-vcluster-latest-images version="0.0.0":
+  {{ASSETS_RUN}} --latest {{ version }} > ./release/images.txt
+
+# Generate the vcluster optional images file
 [private]
-generate-vcluster-images version="0.0.0":
-  go run -mod vendor ./hack/assets/main.go {{ version }} > ./release/vcluster-images.txt
+generate-vcluster-optional-images version="0.0.0":
+  {{ASSETS_RUN}} --optional {{ version }} > ./release/images-optional.txt
```

Comment:
I was originally thinking optional would have latest+optional, but I think this way is good too.